util/stmtsummary/statement_summary.go

```diff
@@ -812,6 +815,7 @@ func (ssElement *stmtSummaryByDigestElement) toDatum(ssbd *stmtSummaryByDigest)
 		formatBackoffTypes(ssElement.backoffTypes),
 		avgInt(ssElement.sumMem, ssElement.execCount),
 		ssElement.maxMem,
+		ssElement.sumDisk,
```

Comment:
It's better to show the max disk usage and average disk usage.

Reply:
Yes, it is better.

Reply:
@djshow832 Please take a look, I do not understand the reason of CI failed...